libopenage/engine.cpp

```diff
@@ -47,6 +47,8 @@ Engine *Engine::instance = nullptr;
 void Engine::create(util::Dir *data_dir, const char *windowtitle) {
 	// only create the singleton instance if it was not created before..
 	if (Engine::instance == nullptr) {
+		// initializa the default actions
+		input::actions::initialize_actions();
```

Comment:
If we do input::actions::initialize_actions() per engine, maybe include it in engine?

If we do global, maybe initialize when static globals are initialized?